lib/tasks/docs.rake

```diff
@@ -178,8 +178,8 @@ namespace :docs do
             "default" => default_value,
             "description" => view_context.render(inline: tag.text)
           }
-
-          f.puts("| `#{tag.name}` | `#{tag.types.join(', ')}` | #{default_value} | #{view_context.render(inline: tag.text)} |")
+          
+          f.puts("| `#{tag.name}` | `#{tag.types.join(', ')}` | #{default_value} | #{view_context.render(inline: tag.text.squish)} |")
```

Comment:
When [multi-lining the tag][0], a newline gets added. This newline causes issues with table formatting:

[0]: https://www.rubydoc.info/gems/yard/file/docs/Tags.md#multi-line-tags

<img width="997" alt="Arguments table with description cell being split into two as a result of a newline" src="https://user-images.githubusercontent.com/16447748/122820495-872e1580-d290-11eb-9e93-5ce9e27ecc83.png">

Added `.squish` to get rid of newlines within tag text. 

<img width="1000" alt="Screen Shot 2021-06-21 at 12 59 43 PM" src="https://user-images.githubusercontent.com/16447748/122820703-c197b280-d290-11eb-9c29-afaf95ef7928.png">